### PR TITLE
feat: handling shipping step in checkout for only BOPIS items

### PR DESCRIPTION
**Pull request by kibo-chandradeeptalaha**

### CODE REVIEW

File: __test__/e2e/integration/checkout/MultiShipCheckoutTemplate/MultiShipCheckoutTemplate.spec.tsx [hunk 1]
```diff
@@ -858,8 +858,10 @@ const handlePaymentStep = async (user: any, checkoutData: Checkout): Promise<Che
 }
 
 const handleReviewStep = async (user: any, checkoutData: Checkout) => {
-  const personalDetailsSection = screen.getByTestId('personal-details')
-  expect(within(personalDetailsSection).getByText(checkoutData.email as string)).toBeVisible()
+  expect(screen.getByRole('heading', { name: 'order-details', level: 2 })).toBeVisible()
+  expect(screen.getByTestId('review-step-component')).toBeVisible()
+  // const personalDetailsSection = screen.getByTestId('personal-details')
+  // expect(within(personalDetailsSection).getByText(checkoutData.email as string)).toBeVisible()
```
Comment (kibo-sushant) [inline __test__/e2e/integration/checkout/MultiShipCheckoutTemplate/MultiShipCheckoutTemplate.spec.tsx]: remove unwanted code

File: components/checkout/PaymentStep/PaymentStep.tsx [hunk 2]
```diff
@@ -481,12 +482,15 @@ const PaymentStep = (props: PaymentStepProps) => {
...
   ])
-
   // handling review order button status (enabled/disabled)
   useEffect(() => {
     if (selectedPaymentBillingRadio) {
+      console.log(stepStatus, isAddingNewPayment)
+
       isAddingNewPayment ? setStepStatusIncomplete() : setStepStatusValid()
     } else {
+      console.log(stepStatus, isAddingNewPayment)
```
Comment (kibo-sushant) [inline components/checkout/PaymentStep/PaymentStep.tsx]: delete console

File: components/checkout/PaymentStep/PaymentStep.tsx [hunk 3]
```diff
@@ -481,12 +482,15 @@ const PaymentStep = (props: PaymentStepProps) => {
     cardFormDetails.cardNumber,
     billingFormAddress.contact.firstName,
   ])
-
   // handling review order button status (enabled/disabled)
   useEffect(() => {
     if (selectedPaymentBillingRadio) {
+      console.log(stepStatus, isAddingNewPayment)
```
Comment (kibo-sushant) [inline components/checkout/PaymentStep/PaymentStep.tsx]: delete console

File: __test__/e2e/integration/checkout/MultiShipCheckoutTemplate/MultiShipCheckoutTemplate.spec.tsx [hunk 4]
```diff
@@ -1021,7 +1023,7 @@ describe('[integration] MultiShipCheckoutTemplate', () => {
       )
     })
 
-    it('should handle the checkout flow', async () => {
+    it.only('should handle the checkout flow', async () => {
```
Comment (Kibo-TriptiV) [inline __test__/e2e/integration/checkout/MultiShipCheckoutTemplate/MultiShipCheckoutTemplate.spec.tsx]: Remove only

File: components/checkout/MultiShippingStep/MultiShippingStep.spec.tsx [hunk 5]
```diff
@@ -204,7 +204,7 @@ const userContextValues = (isAuthenticated: boolean, userId: number) => ({
 
 const TestComponent = ({ param }: { param: any }) => {
   const { data: checkout } = useMultiShipCheckoutQueries({
-    checkoutId: 'test-checkout-id' as string,
+    checkoutId: 'test-checkout-id-1' as string,
```
Comment (Kibo-TriptiV) [inline components/checkout/MultiShippingStep/MultiShippingStep.spec.tsx]: Why we have added test-checkout-id-1, give proper checkout id in spec as well.

File: components/common/ProductItem/ProductItem.tsx [hunk 6]
```diff
@@ -38,6 +38,7 @@ export interface ProductItemProps {
   children?: ReactNode
   width?: string
   subscriptionFrequency?: string
+  hideChangeStore?: boolean
```
Comment (Kibo-TriptiV) [inline components/common/ProductItem/ProductItem.tsx]: Naming conventions should be positive, instead hide use isShowChangeStore.

**Review by Kibo-TriptiV — CHANGES_REQUESTED** [review]
Please check minor review comments

File: __test__/e2e/integration/checkout/MultiShipCheckoutTemplate/MultiShipCheckoutTemplate.spec.tsx (hunk 1 above)
Comment (kibo-chandradeeptalaha) [reply __test__/e2e/integration/checkout/MultiShipCheckoutTemplate/MultiShipCheckoutTemplate.spec.tsx]: Updated

File: components/common/ProductItem/ProductItem.tsx (hunk 6 above)
Comment (kibo-chandradeeptalaha) [reply components/common/ProductItem/ProductItem.tsx]: updated

File: __test__/e2e/integration/checkout/MultiShipCheckoutTemplate/MultiShipCheckoutTemplate.spec.tsx (hunk 1 above)
Comment (kibo-sushant) [reply __test__/e2e/integration/checkout/MultiShipCheckoutTemplate/MultiShipCheckoutTemplate.spec.tsx]: still I can see commented code

File: components/common/ProductItem/ProductItem.tsx [hunk 7]
```diff
@@ -67,6 +68,7 @@ const ProductItem = (props: ProductItemProps) => {
     children,
     width = '25%',
     subscriptionFrequency,
+    isShowChangeStore,
```
Comment (kibo-kevinwatts) [inline components/common/ProductItem/ProductItem.tsx]: I suggest renaming prop to "showChangeStoreLink", its currently a little confusing

**Review by kibo-kevinwatts — APPROVED** [review]

**Review by kibo-sushant — APPROVED** [review]

File: components/common/ProductItem/ProductItem.tsx (hunk 7 above)
Comment (kibo-chandradeeptalaha) [reply components/common/ProductItem/ProductItem.tsx]: Done

**Review by Kibo-TriptiV — APPROVED** [review]

**Pull request merged**